### DISCUSSION TITLE
Welcome to FoxReplace Discussions!

### DISCUSSION THREAD

**Discussion by Woundorf** [Woundorf/foxreplace#323] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

If you have a found a possible bug or have a feature suggestion go to the [issues](https://github.com/Woundorf/foxreplace/issues) section. For questions, general comments, etc. this is the right place.

**Comment by ctrl-qcvh** [comment]
Hey Mark take a look at my misc repository ^_^

**Reply by Woundorf** [reply]
It is truly "misc" indeed. Anyway, for these messages it's better if you send them by email and leave this for discussions, questions, etc. about FoxReplace. I know that I don't always respond to your emails (because sometimes I can't respond at the moment and later I forget), but I always read them.

**Comment by ctrl-qcvh** [comment]
-------- Original Message --------
